spec/unit/libraries/compliance_spec.rb

```diff
@@ -0,0 +1,26 @@
+# encoding: utf-8
+#
+# Cookbook Name:: audit
+# Spec:: compliance
+
+require 'spec_helper'
+require_relative '../../../libraries/collector_classes'
+
+# TODO: can't get this to work: `# ./libraries/collector_classes.rb:251:in `node_info'`
+# need to figure out how to mock the run context
```

Comment:
:( can't figure out how to mock the run context..is there another way around this?